src/time_integration/methods_SSP.jl

```diff
@@ -0,0 +1,188 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
```

Comment:
What's the purpose of these SSP RK methods? Please summarize it briefly here in a comment and add some docstrings below, indicating whether they are experimental or stable.